### PR TITLE
Mock: Fixed that resource properties were not independent of internal state

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.

This change was tested locally with PR https://github.com/zhmcclient/zhmc-ansible-modules/pull/1027.

This PR is needed for PR https://github.com/zhmcclient/zhmc-ansible-modules/pull/1027 and has therefore been set to priority.

### CODE REVIEW

File: zhmcclient_mock/_urihandler.py [hunk 1]
```diff
@@ -829,7 +850,7 @@ def get(cls, method, hmc, uri, uri_parms, logon_required):
         for ucpc in console.unmanaged_cpcs.list(filter_args):
             result_ucpc = {}
             for prop in cls.returned_props:
-                result_ucpc[prop] = ucpc.properties[prop]
+                result_ucpc[prop] = prop_copy(prop_copy(ucpc.properties[prop]))
```
Comment (EdGue42) [inline zhmcclient_mock/_urihandler.py:853]: why copy the copy?

File: changes/1583.fix.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Mock: Fixed that resource properties returned from zhmcclient mock support
+were not independent of the internal resosurce object's state. For properties
```
Comment (simon-spinner) [inline changes/1583.fix.rst:2]:
```suggestion
were not independent of the internal resource object's state. For properties
```

File: zhmcclient_mock/_urihandler.py (hunk 1 above)
Comment (andy-maier) [reply zhmcclient_mock/_urihandler.py:853]: oops. one editor change too much. Thanks!

File: changes/1583.fix.rst (hunk 2 above)
Comment (andy-maier) [reply changes/1583.fix.rst]: Fixed.

**Pull request merged**